pkg/machinery/config/contract.go

```diff
@@ -147,3 +147,8 @@ func (contract *VersionContract) ApidExtKeyUsageCheckEnabled() bool {
 func (contract *VersionContract) APIServerAuditPolicySupported() bool {
 	return contract.Greater(TalosVersion1_2)
 }
+
+// KubeletEnableManifestsDirectoryFlagSupported returns true if kube-apiserver custom audit policy is supported.
+func (contract *VersionContract) KubeletEnableManifestsDirectoryFlagSupported() bool {
```

Comment:
```suggestion
func (contract *VersionContract) KubeletManifestsDirectoryDisabled() bool {
```

I thtink this communicates the intention better - we disable the manifests directory.